metrics/grafana/tikv_details.json

```
@@ -422,9 +422,36 @@
               "expr": "sum(tikv_store_size_bytes{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", type=\"used\"}) by (instance)",
               "format": "time_series",
               "intervalFactor": 2,
-              "legendFormat": "{{instance}}",
+              "legendFormat": "{{instance}}--used",
```

Referring pervious settings, using single '-' is preferable.
```suggestion
              "legendFormat": "{{instance}}-used",
```

---

fixed